pkg/pillar/docs/volumemgr.md

```diff
@@ -65,17 +80,56 @@ When volumemgr starts it finds existing volumes on disk and publishes those (for
 
 When volumemgr gets a request for a volume, it first looks whether it matches an existing volume. That might exist in the above "unknown" agentScope collection, in which case it is promoted to the agentScope for the particular requester. In this case its work is complete.
 
-Otherwise (for the OriginTypeDownload), the volumemgr looks for any existing already verified blob in VerifyImageStatus and PersistImageStatus. If one is found, it ensures that there is a corresponding VerifyImageConfig and PersistImageConfig to retain a reference count on the verified image.
+Otherwise (for the `OriginTypeDownload`), volumemgr looks for any existing already verified blob in `VerifyImageStatus` and
+`PersistImageStatus`. If one is found, it ensures that there is a corresponding `VerifyImageConfig` and `PersistImageConfig`
+to retain a reference count on the verified image.
```

Comment:
Extra CR before this line?

Reply:
markdown parses it fine. If you look at the file rendered [here](https://github.com/lf-edge/eve/blob/af38bd2b7fcf716521946b602382c92b511076e9/pkg/pillar/docs/volumemgr.md), it looks ok.